Enterprise-CMCS/macpro-mdct-mfp
# Move getTemplate to s3-lib, in order to delete an obsolete ETL

### Description
See title. The `populateTemplatesTable` migration was written for MCR, which previously had form templates and field data in a 1:1 relationship. The migration was necessary to break that relationship into 1:many in MCR, but MFP has no use for it.

There was one function in this migration which was used elsewhere in MFP. I moved that function to s3-lib, which feels like a fitting home.

Deleting this migration (and its tests, which covered it rather poorly) has the effect of bumping test coverage in app-api by about 1%.

### Related ticket(s)
CMDCT-3121

---
### How to test
Ensure reports can still be created and loaded. This will confirm that `getTemplate` is working properly from its new location, and that I didn't break anything else in s3-lib.

### Important updates
n/a

---
### Author checklist
